templates/email/design.js

```diff
@@ -84,28 +95,24 @@ module.exports = cx.design
       .withGroupId('images-hRO9mw')
       .withLabel('Bilder')
       .withContentElements(
-        require('@bsi-cx/design-standard-library-email/content-elements/base/image'),
-        require('./content-elements/base/img-66'),
-        require('./content-elements/base/img-50'),
-        require('./content-elements/base/img-33'),
+        require('./content-elements/base/img-with-style').withMinVersion(Version.CX_23_2),
+        require('@bsi-cx/design-standard-library-email/content-elements/base/image').withArchivedMinVersion(Version.CX_23_2),
```

Comment:
I would have expected to either use the `image` element from the standard library or to implement the 'img-with-style' element in the library